ledger/participant-integration-api/src/main/scala/platform/store/dao/events/LfValueTranslation.scala

```diff
@@ -264,12 +255,17 @@ final class LfValueTranslation(
     ValueSerializer.deserializeValue(algorithm.decompress(new ByteArrayInputStream(value)))
 
   def enricher: ValueEnricher = {
-    // Note: LfValueTranslation is used by JdbcLedgerDao for both serialization and deserialization.
-    // Sometimes the JdbcLedgerDao is used in a way that it never needs to deserialize data in verbose mode
-    // (e.g., the indexer, or some tests). In this case, the enricher is not required.
     enricherO.getOrElse(
       sys.error(
-        "LfValueTranslation used to deserialize values in verbose mode without a ValueEnricher"
+        "LfValueTranslation used to deserialize values in verbose mode without an Engine"
+      )
+    )
+  }
+
+  def engine: Engine = {
+    engineO.getOrElse(
+      sys.error(
+        "LfValueTranslation used to deserialize values in verbose mode without an Engine"
       )
     )
   }
```

Comment:
🙏 
```suggestion
  def engine: Engine = 
    engineO.getOrElse(
      sys.error(
        "LfValueTranslation used to deserialize values in verbose mode without an Engine"
      )
    )
```

Reply:
I hate odd braces too, but still sometimes miss it ;) thanks for pointing it out!